packages/tc-ui/src/lib/components/buttons.jsx

```diff
@@ -7,6 +7,7 @@ const getButtonClasses = extraClasses =>
 
 const EditButton = props => (
 	<a
+		data-testid="edit"
```

Comment:
is there some way we can hook onto existing data-attributes, or if we need data attributes/classes to pick out these buttons in tests, just use non test names e.g. `data-button-type="edit"`